src/config.js

```diff
@@ -21,7 +21,7 @@
 import browser from 'bowser';
 
 const PREFIX =
-  browser.webkit ? '-webkit-' :
+  browser.webkit || browser.blink ? '-webkit-' :
```

Comment:
`.blink` was added in bowser 1.2.0, so you might want to raise the package.json dependency for it.

Ref:
https://github.com/ded/bowser/releases/tag/1.2.0